### PR TITLE
Adding query summary & results per page component

### PR DESCRIPTION
**Pull request by bala-one**
I have made the following updates

- Add component Query Summary
- Add component Results Per Page
- VS Code tasks for easy start and debug
- Prettier format task
- Some essential style updates

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
```
Comment (jpmarceau) [inline .vscode/launch.json:1]: I don't think it makes sense to include this task here, as most users won't have edge installed, and neither do I think we want to explain to people how to set up the debugger to work in this case. Let's keep the tutorial simple 🙂

File: src/App.tsx [hunk 2]
```diff
@@ -32,14 +36,19 @@ function App() {
         <div className="search-section">
           <SearchBox controller={searchBox} />
         </div>
+
         <div className="main-section">
           <div className="facet-section column">
             <Facet controller={facet} title="Source" />
           </div>
           <div className="results-section column">
+            <div className="query-section">
```
Comment (jpmarceau) [inline src/App.tsx:45]: Thank you for the suggestion, but I think it's best not to add new components. This tutorial app was kept very simple on purpose, to ease the learning experience.

File: src/App.css [hunk 3]
```diff
@@ -25,6 +25,11 @@
 
 .facet ul {
   text-align: left;
+  padding-left: 10px;
```
Comment (jpmarceau) [inline src/App.css:28]: Actually I think I would rather not modify this CSS. In this case as well, I think it's better to keep it very minimal so learners understand that Headless only takes care of the application state, and that users have to handle the CSS themselves.

File: .vscode/tasks.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (jpmarceau) [inline .vscode/tasks.json:1]: We do not require learners to use vscode for this tutorial, so I don't think it makes sense to include this vscode task in the repo.

**Review by jpmarceau — CHANGES_REQUESTED** [review]
Hi Bala,
Apologies for the delayed response.
Thanks for the suggestions, however I think we should not implement most of them.
Do you know that this repo is used for a [tutorial](https://levelup.coveo.com/learn/courses/headless-tutorial)? I just made that a little bit clearer in the Readme by including a link. It was not as obvious before.
Accordingly, we would like to keep this project as simple as possible to help people focus on the tutorial lessons.
That's why there are so few components in the app, and why there is so little styling. It's also why there are so little tools like vscode tasks or npm scripts.
Overall, I think it would only make sense to keep your formatting npm script.

**Comment by bala-one** [comment]
Thank you for the review comments.

**Pull request closed without merging**